packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -74,6 +74,9 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
       buildDir: buildDirectory,
       sourceDir: windowsProject.cmakeFile.parent,
     );
+    if (visualStudio.displayVersion?.startsWith('17.1.') == true) {
```

Comment:
an imminently merging lint will require this to be written as `?? false` rather than `== true`

Reply:
Thanks, changed.